plugins/api-docs/src/components/GraphQlDefinitionWidget/GraphQlDefinition.tsx

```diff
@@ -46,17 +53,46 @@ type Props = {
 
 export const GraphQlDefinition = ({ definition }: Props) => {
   const classes = useStyles();
+  const { entity } = useEntity();
```

Comment:
This is a major change in contract for the component. It used to be possible to render it outside of an entity page, and now you can't. If people were using this component standalone to render a graphql definition, their Backstage will suddenly break after upgrading.

I suggest adding the endpoint URL to its props instead, as an optional string. Then leave it up to the surroundings to supply if any way they see fit, if they want to